Sources/BSON/BSONError.swift

```diff
@@ -42,6 +42,22 @@ public enum BSONError {
     }
 }
 
+extension DecodingError {
+    /// Standardize the errors emitted by BSONValue initializers.
+    internal static func _extendedJSONError(
+        keyPath: [String],
+        debugDescription: String
+    ) -> DecodingError {
+        let debugStart = keyPath.joined(separator: ".") +
+            (keyPath == [] ? "" : ": ")
+        print(debugStart + debugDescription)
```

Comment:
print statement here needs to be removed

Reply:
Looks good to me as well, just needs this print removed.

Reply:
done